lcm-python/meson.build

```diff
@@ -0,0 +1,40 @@
+pymod = import('python')
+py_installation = pymod.find_installation()
+python_site = py_installation.get_path('platlib').replace(py_installation.get_path('data') / '', '')
+python_include = py_installation.get_path('include')
+
+lcm_python_sources = ['module.c', 'pyeventlog.c', 'pylcm.c', 'pylcm_subscription.c']
+
+if windows
+  lcm_python_lib = library('lcm-python', lcm_python_sources,
+    dependencies : lcm_static_lib_dep,
+    include_directories : include_directories(python_include),
+    install_dir : python_site + '/lcm',
+    name_suffix : '.pyd')
+elif host_machine.system() == 'darwin' #macos
+  lcm_python_lib = library('lcm-python', lcm_python_sources,
+    dependencies : lcm_static_lib_dep,
+    include_directories : include_directories(python_include),
+    install_dir : python_site + '/lcm',
+    link_args : '-undefined dynamic_lookup -Wl,-no_fixup_chains')
+else
+  lcm_python_lib = library('lcm-python', lcm_python_sources,
+    dependencies : lcm_static_lib_dep,
+    include_directories : include_directories(python_include),
+    install_dir : python_site + '/lcm')
+endif
```

Comment:
You'll want [a Python extension module](https://mesonbuild.com/Python-module.html#extension_module) rather than `library` here

Reply:
Switched to extension_module and set up install: 7ebd7891cc0ffeea7d057de18968b8eff8985c89